### PR TITLE
Do fixture data release hop

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Its a catch 22, Cant route the STM until Mozart shows a render, Mozart wont show a render until the STM

**Code changes:**

- Makes frontpage fixture data available in simorgh (for internal use)
- Makes frontpages use their own Base URL which will be localhost (itself) until we route test and stage.

**The steps:**
1) Simorgh frontpages request data from itself for the server render (fixture)
2) We see a server render on mozart test
3) We route the STM on test
4) We update `test.env` to use `https://www.test.bbc.com`
5) We see server render on mozart live
6) We route the STM on stage
7) We update `live.env` to use `https://www.stage.bbc.com`
8) We stop simorgh serving frontpage fixture data

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by dr3** [comment]
Ive raised https://github.com/bbc/simorgh-infrastructure/issues/494 to fix this issue in future

File: envConfig/live.env [hunk 1]
```diff
@@ -1,4 +1,5 @@
 SIMORGH_BASE_URL=https://www.bbc.com
+SIMORGH_FRONTPAGE_BASE_URL=http://localhost:7080
```
Comment (12) [inline envConfig/live.env:2]: Is this correct for live?

File: envConfig/test.env [hunk 2]
```diff
@@ -1,4 +1,5 @@
 SIMORGH_BASE_URL=https://www.test.bbc.com
+SIMORGH_FRONTPAGE_BASE_URL=http://localhost:7080
```
Comment (12) [inline envConfig/test.env:2]: Same Q

File: envConfig/live.env (hunk 1 above)
Comment (12) [reply envConfig/live.env:2]: lol just read the description of this pr, stupid me

**Review by j-pendlebury — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]
👍

**Comment by david-boydell** [comment]
Not much for test to do here, this is ready for merge.

**Pull request merged**